create-publication-from-doi/src/test/java/no/unit/nva/doi/fetch/MainHandlerTest.java

```diff
@@ -107,17 +108,17 @@ public void testOkResponse()
 
         mainHandler.handleRequest(mainHandlerInputStream(), output, context);
 
-        GatewayResponse gatewayResponse = objectMapper.readValue(output.toString(), GatewayResponse.class);
+        GatewayResponse<?> gatewayResponse = objectMapper.readValue(output.toString(), GatewayResponse.class);
         assertEquals(SC_OK, gatewayResponse.getStatusCode());
-        assertTrue(gatewayResponse.getHeaders().keySet().contains(CONTENT_TYPE));
-        assertTrue(gatewayResponse.getHeaders().keySet().contains(MainHandler.ACCESS_CONTROL_ALLOW_ORIGIN));
+        assertTrue(gatewayResponse.getHeaders().containsKey(CONTENT_TYPE));
+        assertTrue(gatewayResponse.getHeaders().containsKey(MainHandler.ACCESS_CONTROL_ALLOW_ORIGIN));
         Summary summary = objectMapper.readValue(gatewayResponse.getBody(), Summary.class);
         assertNotNull(summary.getIdentifier());
     }
 
     @Test
     public void processInputThrowsIllegalStateExceptionWithInternalCauseWhenUrlToPublicationProxyDoesIsNotValid()
```

Comment:
rename to `processInputThrowsIllegalStateExceptionWithInternalCauseWhenUrlToPublicationProxyIsNotValid`